src/Connector.SqlServer/Connector/SqlServerConnector.cs

```diff
@@ -173,8 +173,13 @@ public async Task VerifyExistingContainer(ExecutionContext executionContext, Str
             {
                 var upgrade = _features.GetFeature<IUpgradeTimeStampingFeature>();
                 var config = await base.GetAuthenticationDetails(executionContext, stream.ConnectorProviderDefinitionId.Value);
-
                 await upgrade.VerifyTimeStampColumnExist(_client as ISqlClient, config, stream);
+
```

Comment:
not sure if this method is called on something other than stream creation.

Reply:
Since the stream will be "created", whenever we're starting the server, it will be called.